Post release doc update (#817)
oamg/leapp: docs/source/el7toel8/actor-rhel7-to-rhel8.md

reporting.Remediation(playbook=<link_to_playbook>)
```

**Available Groups**

> Following Groups have been originally known as **Tags**:

Reviewer: ```suggestion
The following groups were originally known as **Tags**:
```

```
'accessibility', 'authentication', 'boot', 'communication', 'drivers', 'email', 'encryption',
'filesystem', 'firewall', 'high availability', 'kernel', 'monitoring', 'network', 'OS facts',
'post', 'python', 'repository', 'sanity', 'security', 'selinux', 'services', 'time management',
'tools', 'upgrade process'
```

> Following groups have been originally known as **Flags**:

Reviewer: ```suggestion
The following groups were originally known as **Flags**:
```

```
'failure', 'inhibitor'
```

> The **failure** Group is recommended to be used when the report is related to

Reviewer: ```suggestion
The **failure** group is recommended when the report is related to
```

a command or other action failure.

> In case of need of additional report groups, please open a GH issue or a PR,

Reviewer: ```suggestion
If you need additional report groups, please open a GH issue or a PR,
```
